index.html

```diff
@@ -226,76 +226,160 @@ <h1>The Parameter Registry</h1>
-          <td>
-TBD
-          </td>
-          <td>
-            <div>DESCRIPTION_GOES_HERE</div>
-            <div><a href="">SPECIFICATION_GOES_HERE</a></div>
-            <pre>
-"@context": [
-  "https://www.w3.org/ns/did/v1",
-  "https://CONTEXT_GOES_HERE/v1"
-]
-            </pre>
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </section>
-
-
     <section>
-      <h1>The Value Registry</h1>
-
-      <p>
-  This table summarizes the globally registered values that can be used as
-  property values in a DID Document.
-      </p>
+      <h2><dfn data-lt="publicKey" data-dfn-type="dfn" id="publicKey">publicKey</dfn></h2>
+      <p>An array of public keys. A public key is a verification method. Public keys are used for digital signatures, encryption and other cryptographic operations, which in turn are the basis for purposes such as authentication.</p>
+      
+      <pre class="example">
+        {
+          "id": "did:example:123",
+          "publicKey": [
+            {
+              "id": "did:example:123#ZC2jXTO6t4R501bfCXv3RxarZyUbdP2w_psLwMuY6ec",
+              "type": "Ed25519VerificationKey2018",
+              "controller": "did:example:123",
+              "publicKeyJwk": {
+                "crv": "Ed25519",
+                "x": "o3v3B4wU7LPewgr3iQDSc9LGzJKdS0R_698dUXtBXX8",
+                "kty": "OKP",
+                "kid": "ZC2jXTO6t4R501bfCXv3RxarZyUbdP2w_psLwMuY6ec"
+              }
+            },
+            {
+              "id": "did:example:123#WjKgJV7VRw3hmgU6--4v15c0Aewbcvat1BsRFTIqa5Q",
+              "type": "EcdsaSecp256k1VerificationKey2019",
+              "controller": "did:example:123",
+              "publicKeyJwk": {
+                "crv": "secp256k1",
+                "x": "NtngWpJUr-rlNNbs0u-Aa8e16OwSJu6UiFf0Rdo1oJ4",
+                "y": "qN1jKupJlFsPFc1UkWinqljv4YE0mq_Ickwnjgasvmo",
+                "kty": "EC",
+                "kid": "WjKgJV7VRw3hmgU6--4v15c0Aewbcvat1BsRFTIqa5Q"
+              }
+            }
+          ]
+        }
+      </pre>
+
+      <h4>Definitions</h4>
+      <table class="simple" style="width: 100%;">
+        <thead>
+          <tr>
+            <th>JSON Schema</th>
+            <th>JSON-LD</th>
+            <th>CBOR</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <a href="https://w3c.github.io/did-core-registry/schemas/publicKey.json">JSON Schema</a>
+            </td>
+            <td>
+              <a href="https://w3c.github.io/did-core-registry/contexts/did-core-latest.jsonld">JSON-LD</a>
+            </td>
+            <td>
+              CBOR
+            </td>
+          </tr>
+        </tbody>
+      </table>
 
-      <table class="simple">
+      <section>
+      <h3><dfn data-lt="Ed25519VerificationKey2018" data-dfn-type="dfn" id="Ed25519VerificationKey2018">Ed25519VerificationKey2018</dfn></h3>
+
+      <p>A Ed25519 publicKey.</p>
+      
+      
+      <pre class="example">
```

Comment:
Same comment as above, use simpler publicKeyBase58 encoding.

Reply:
Why would we use base58 encoding when JWK is a valid encoding for all key types?

Reply:
Because the JWK key expression mechanism exposes too much to the developer, it is also overly complicated for expressing an Ed25519 key, and we should avoid leading with complicated examples. Key material should be expressed as one opaque value, not multiple opaque values that can be easily jumbled by developers that don't know any better.

That said, I just went off and reviewed the latest node.js and OpenSSL documentation for loading private keys -- we used to pick this format because you could fairly easily feed it directly to crypto libraries, avoiding JWK libraries entirely... now I'm seeing that the latest node requires PEM encoding... https://nodejs.org/api/crypto.html#crypto_crypto_createprivatekey_key

What am I missing @dlongley @dmitrizagidulin @mattcollier? I thought we were feeding publicKeyBase58 directly to these libs after base58-btc decoding? Perhaps this is because we also wanted this value to match the cryptonyms in Veres One and going to PEM was trivial?